content/css-blog.md

```diff
@@ -0,0 +1,36 @@
+---
+title: When CSS seems scary
+image: /images/writing.jpg
+imageMeta:
+  attribution:
+  attributionLink:
+featured: true
+authors:
+  - anne-greeth-schot-vanherwijnen
+date: Wed Oct 6 2021
+tags:
+  - css
+  - tech
+  - beginner
+---
+
+In this blogpost I want to show you how I tackle CSS. There are so many CSS rules, so many properties and options that it might feel overwhelming. I would recommend you to start small. 
```

Comment:
```suggestion
In this blogpost I want to show you how I tackle CSS. There are so many CSS rules, so many properties and options that it might feel overwhelming. I would recommend that you start small. 
```